metrics/grafana/tikv_details.json

```diff
@@ -4654,6 +4654,232 @@
             "align": false,
             "alignLevel": null
           }
+        },
+        {
+          "aliasColors": {},
+          "bars": true,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "${DS_TEST-CLUSTER}",
+          "fieldConfig": {
+            "defaults": {},
+            "overrides": []
+          },
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 12,
+            "x": 0,
+            "y": 68
+          },
+          "hiddenSeries": false,
+          "id": 23763572784,
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": false,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "7.5.11",
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [
+            {
+              "$$hashKey": "object:80",
+              "alias": "/.*/",
+              "stack": "A"
+            }
+          ],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "sum(rate(tikv_storage_rocksdb_perf{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", metric=\"block_read_time\"}[1m])) by (req)",
+              "hide": false,
+              "interval": "",
+              "legendFormat": "{{req}}",
```

Comment:
It's good to have, but I also have some worries.

I'm thinking if `request_source` should be combined into the same panel. Will it cause too many series and let the graph unreadable?

And `request_source` * `command type` can be very large, could it be a problem that it produces too many label combinations and cause size bloat.

My initial thought is creating a separate similar graph, but the series are the request source.  This is definitely less informative, but maybe enough for most cases...

Reply:
> `My initial thought is creating a separate similar graph, but the series are the request source. ` 
This looks like a better usage that avoids the data size bloat problem.  
@you06 What do you think?

Reply:
I do have the same concern, and my thought is to reduce the kinds of `request_source`. But `request_source` * `command type` would be a large amount whatever. A separate similar graph makes sense and it'll produce only `request_source` + `command type` metrics.

> but maybe enough for most cases...

With data from two dimensions, it can already locate many problems.